Firestore/Example/Tests/SpecTests/FSTSyncEngineTestDriver.h

```diff
@@ -311,6 +313,28 @@ typedef std::unordered_map<firebase::firestore::auth::User,
 /** The current user for the FSTSyncEngine; determines which mutation queue is active. */
 @property(nonatomic, assign, readonly) const firebase::firestore::auth::User &currentUser;
 
+/**
+ * The number of snapshots-in-sync calls that have been run.
+ */
+@property(nonatomic, readonly) int snapshotsInSyncEvents;
+
+- (void)incrementSnapshotsInSyncEvents;
+
+- (void)resetSnapshotsInSyncEvents;
+
+/**
+ * Adds a snapshots-in-sync listener from the event manager the spec test vector.
```

Comment:
"from" should probably be "to".

Also, this sentence is missing words. Maybe you meant "event manager *and* the spec test vector"? However, consider dropping "spec test vector" instead: I have no idea what that even is.

If you do want to keep it, describe what you're doing using higher-level terminology: aim at someone who hasn't implemented the feature describing the behavior in standalone terms. Only describe specific changes to the implementation when it's important or subtle.

If you were talking about this adding to `_snapshotsInSyncListeners`, then I'd rephrase as:

> Adds a snapshots-in-sync listener to the event manager and keeps track of it so that it can be easily removed later.